data/campaigns/Eastern_Invasion/scenarios/03_An_Unexpected_Appearance.cfg

```diff
@@ -1,167 +1,173 @@
     [side]
         side=1
         controller=human
-        {GOLD 200 180 160}
+        gold=150
         team_name=wesnothians
         user_team_name= _ "Wesnothians"
         {FLAG_VARIANT loyalist}
-
-        # wmllint: recognize Gweddry
         {CHARACTER_STATS_GWEDDRY}
     [/side]
-    # wmllint: validate-on
 
     [side]
         side=2
         controller=ai
-        recruit=Skeleton,Skeleton Archer,Skeleton Rider,Soulless
-        {GOLD 70 85 100}
-        {INCOME 1 2 3}
+        recruit=Walking Corpse,Soulless,Ghoul,Skeleton Rider
+        {GOLD 20 60 100}
+        {INCOME -2 0 4}
         team_name=undead
         user_team_name= _ "Undead"
         {FLAG_VARIANT undead}
-
         type=Dark Sorcerer
         id=Mal-Skraat
         name= _ "Mal-Skraat"
         canrecruit=yes
-
         facing=ne
         [modifications]
             {TEAM_COLOR_OVERRIDE () darkred}
+            {TRAIT_INTELLIGENT}
+            {TRAIT_STRONG}
         [/modifications]
-
         [ai]
-            recruitment_pattern=scout,scout,fighter,fighter,archer
-        [/ai]
-
-        [ai]
-            recruitment_pattern=fighter,archer,scout,scout
-            aggression=0.6
-            grouping=no
-            [goal]
-                [criteria]
-                    id=Gweddry
-                [/criteria]
-                value=5
-            [/goal]
+            scout_village_targeting=0.8
+            leader_value=0
         [/ai]
     [/side]
+    {STARTING_VILLAGES 2 9}
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 2 "Vampire Bat" 1}
+    {RECRUIT_UNIT_VARIATIONS 2 "Walking Corpse" none,none,mounted,mounted}
+    {RECRUIT_UNIT_VARIATIONS 2 "Soulless" none,none,mounted,mounted}
+    {SILENTLY_LIMIT_LEADER_MOVES 2 1}
 
     [side]
         side=3
         controller=ai
         recruit=Walking Corpse
-        {GOLD 10 20 30}
+        {GOLD 0 0 8}
         team_name=undead
         user_team_name= _ "Undead"
         {FLAG_VARIANT undead}
-
         type=Dark Adept
         id=Mal-Tar
         name= _ "Mal-Tar"
         canrecruit=yes
-
         facing=sw
-        experience=24
-
+        experience=11
+        [modifications]
+            {TRAIT_INTELLIGENT}
+            {TRAIT_RESILIENT}
+        [/modifications]
         [ai]
             aggression=1.0
+            leader_aggression=1.0
+            leader_value=0
         [/ai]
+#ifndef EASY
+        {GENERIC_UNIT 3 (Walking Corpse) 25 11}
+#endif
     [/side]
+    {SILENTLY_LIMIT_LEADER_MOVES 3 1}
+    {FORCE_CHANCE_TO_HIT id=Gweddry,Dacyn side=3 100 ()} # make sure you can't lose from bad luck
 
     [side]
         side=4
         controller=ai
-        recruit=Vampire Bat,Skeleton,Skeleton Archer,Ghoul
-        {GOLD 70 85 100}
-        {INCOME 2 3 4}
+        recruit={ON_DIFFICULTY (Vampire Bat) (Vampire Bat,Blood Bat,Ghost) (Vampire Bat,Blood Bat,Ghost)} # on easy, she might never recruit if she tries to save money
+        {GOLD 13 35 60} # needs at least 13 gold to get her first vampire bat
```

Comment:
Possible alternatives:
- `{13 36 60}`
- `{13 39 65}`
- `{15 40 65}`

Reply:
`{13 39 65}` sounds fine to me